variables.tf

```diff
@@ -1,9 +1,9 @@
-variable env_code {
-    type = string
+variable "env_code" {
+  type = string
 }
 
-variable vpc_cidr {}
+variable "vpc_cidr" {}
 
-variable private_cidr {}
+variable "private_cidr" {}
 
-variable public_cidr {}
\ No newline at end of file
+variable "public_cidr" {}
```

Comment:
Each file should end with a newline. Please add a newline to the end of this file